### DISCUSSION TITLE
Change colour of <p> tag via tailwind.config?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [tailwindlabs/tailwindcss#9389] [Help]
Hi

I'm using the typography plugin, and I can extend the styles of that via `tailwind.config` 

But how do I do the same for HTML outside of `prose`?

The `p` tag is given a default colour, is there a way to overide this globally? Without adding a` text-gray-900 `class to body or the `p` tag directly, or adding styles to the `base `layer in the `css` files?

**Comment by wongjn** [comment]
You have described and excluded most of the methods available but you could [write a custom plugin and use `addBase()`](https://tailwindcss.com/docs/plugins#adding-base-styles) to add the color to `<p>` tags. In your `tailwind.config.js`:

```js
const plugin = require('tailwindcss/plugin');

module.exports = {
  // …
  plugins: [
    plugin(({ addBase, theme }) => {
      addBase({
        p: {
          // Any color value you like, doesn't have to use `theme()`,
          // you can set a hard-coded value like '#012345' as a string.
          color: theme('colors.slate.500'),
        },
      });
    }),
  ],
};
```
